Editors/README.md

```diff
@@ -22,56 +22,34 @@ The [Swift for Visual Studio Code extension](https://marketplace.visualstudio.co
 
 ## Sublime Text
 
-Before using SourceKit-LSP with Sublime Text, you will need to install the LSP package from Package Control. To configure SourceKit-LSP, open the LSP package's settings. The following snippet should be enough to get started with Swift.
+Before using SourceKit-LSP with Sublime Text, you will need to install the LSP and LSP-SourceKit packages from Package Control. And you're ready to go if you have `xcrun` in the `$PATH`.
 
-You will need the path to the `sourcekit-lsp` executable for the "command" section.
+To configure SourceKit-LSP additionally, open the SourceKit-LSP package's settings by typing in command palette `Preferences: LSP-SourceKit Settings`. The following snippet should be enough to get started with Swift and Objective-C/C++ and the custom path to `sourcekit-lsp` executable.
```

Comment:
1. Fair point, fixed.
2. I'd rather leave this one to @ahoppen decision.
3. This one is interesting. AFAIK `xcrun` is a part of xcodebuild tools only, thus it provided on macOS only, so in this case, since ST is focused on being crossplatfrom, maybe to change the default setup from `xcrun sourcekit-lsp` up to `sourcekit-lsp` could be a better choice. Anyway, it was discussed just right [here](https://github.com/apple/sourcekit-lsp/pull/1184#discussion_r1571461864), and Alex insisted to keep this line in anyway.

Reply:
If `sourcekit-lsp` is a better command than `xcrun sourcekit-lsp` then this should be updated in the `LSP-SourceKit` package and then this override could be removed here.

Reply:
Again it's on Alex duty to decide

Reply:
> If sourcekit-lsp is a better command than xcrun sourcekit-lsp then this should be updated in the LSP-SourceKit package and then this override could be removed here.

I agree, it would be better to update that package. Also, do we even need to specify the `selector` here or could that also be sunk down into the `LSP-SourceKit` package, in which case no additional configuration would be necessary?

Reply:
The LSP-SourceKit package by default only enables server for swift files. Does it make sense to enable it for c/c++/objc by default? I guess it could be problematic to have it running in projects that are not really swift projects?

Reply:
Oh, I see. The VS Code extension specifies support for C/C++/ObjC/ObjC++ by default, so I think that is a default that we can use here as well. That way, they are aligned.

Reply:
> The VS Code extension specifies support

It's some kind of similar to suggesting a feature to macOS team by referencing to Windows design decision 😀 It could work somewhat with zed though.

Anyway I'd rather agree with that, there's a loosing chance that some one who working on plain c/c++ codebase would occasionally install and turn on globally LSP-SourceKit that'll brings to a lsp servers conflict on an ST side.

So if @rchl has nothing against that, I can provide another LSP-SourceKit that (a) extends its scope to the list above and (b) replaces a given `xcrun sourcekit-lsp` command to `/usr/bin/sourcekit-lsp` or whatever that we would decide in the thread above.

Reply:
I think that replacing `xcrun sourcekit-lsp` with `sourcekit-lsp` should be done in the LSP-SourceKit package. At least as long as it's tested on mac.

As for enabling for other file types... I suppose it could be a double edged sword. On one hand it would potentially enable server in projects that don't use swift. On the other hand, not enabling for other file types probably makes the server less effective in swift projects.

So one way of handling that could be to:
1. Disable LSP-SourceKit server by default (have `enabled: false` in its default settings
2. Add instructions in its readme that if the user wants to enable it, he/she has to run "LSP: Enable language server in project" from the command palette.
So users would explicitly opt-in to it per-project.

Reply:
That sounds like a great idea to me.

Also @rchl: Thanks for engaging in this discussion and sharing your knowledge of ST packages. That’s really valuable. 🙏🏽